jax/_src/lax/convolution.py

```diff
@@ -747,27 +746,6 @@ def _reshape_axis_out_of(src, size1, x):
   return lax.reshape(x, shape)
 
 
-def _check_conv_shapes(name, lhs_shape, rhs_shape, window_strides):
```

Comment:
This looks like a useful function to have. Shouldn't it be used in say `lax.conv`?

Reply:
Maybe it could be used? I can't find the last time this function was called in the codebase, so it would take some work to ensure the logic is correct in all corner cases.